vignettes/subsetting_tables.Rmd

```diff
@@ -168,3 +168,36 @@ Note the return value of `cell_values` is always a list even if you specify a pa
 ```{r}
 cell_values(tbl2, c("RACE", "ASIAN", "AGE",  "Mean"), c("ARM", "A: Drug X", "SEX", "F"))
 ```
+
+## Rows and cell values alignments
```

Comment:
I think this section overall is good (with the caveat in the comment below) but I don't think it fits well in the subsetting table vignette (if I were looking for it I wouldn't think to look there), so please move it there (until we have a specialized 'modifying appearance' vignette, which we probably should, that would cover indent modifier,  alignment, formatting, table inset, section and hsep dividers)

Reply:
So I start creating a "Modifying appearance" vignette with this for a start? I can also add something about formats and na_str. THis would possibly close #366

Reply:
I would call it customizing appearance, rather than modifying, because these things can be declared up front in a layout (`basic_table(inset=2)`, `analyze(lyt, afun = function(x) rcell(6, align = "right"))`, etc)

Reply:
of course! Done ;)